src/schema/types/user.graphql

```diff
@@ -1,4 +1,21 @@
 type User {
   id: ID!
   name: String!
+  email: String!
+}
+
+input UserInput {
+  name: String!, 
```

Comment:
No commas needed. This is not a Javascript object but GraphQL notation.

Reply:
Done